components/Templates/page_layout.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Page Layout</title>
+  <link rel="stylesheet" href="#">
+  <link rel="stylesheet" href="#">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+  <div class="wrap">
+    <div class="background-header"></div>
+    <aside>
```

Comment:
I would consider pulling the `<aside>` into the `container`. This would allow you to use `display: flex` on the container to get the aside positioned correctly side-by-side with your content without having to use the `padding-right` approach.

Here's a great post about flex and its usage: https://css-tricks.com/snippets/css/a-guide-to-flexbox/